### ISSUE TITLE
Can't upload files to specific directory from node server

### ISSUE THREAD

**Issue by bnjm** [archilogic-com/3dio-js#59]
Trying to upload files to a specific directory on storage.3d.io from node.js with the following snippet:

```
const io3d = require('3dio')
const file = new Buffer(['Hello World'])

io3d.config({
  secretApiKey: '...'
})

io3d.storage
  .put(file, {
    key: '/my-folder-name/my-file-name.txt'
  })
  .then(console.log)
```

It returns `Unhandled rejection You have to be logged in to store files in a specific location`. However auth.logIn does not work in node environments #54. 

Are specific directory file uploads supported from non-browser environments?

**Comment by frederic-schwarz** [comment]
https://3d.io/docs/api/1/storage.html#upload-file-to-a-specific-location-

**Comment by bnjm** [comment]
Things I also tried: 

`key: '/{{userId}}/...'`

-> `Using key parameter with template syntax requires authentication.`

`key: '/...fetching actual userID/...'/`

-> `Unhandled rejection You have to be logged in to store files in a specific location`

**Comment by AVGP** [comment]
This does require a backend fix and a new release of this library.

The frontend part of the fix is in 987f882d66b12c825a7725453dfdb3b4515d6737 which basically allows to use `{{userId}}` with secret keys, the backend fix will be released today.
